app/views/results/common/_peer_reviews.html.erb

```diff
@@ -29,7 +30,7 @@
 
   <% if @current_role.student? %>
     <%= link_to t('peer_reviews.view_ta_result'),
-                view_marks_assignment_submission_result_path(assignment.id, submission.id,
+                view_marks_course_assignment_submission_result_path(@current_course.id, assignment.id, submission.id,
```

Comment:
This route helper method is not right. Result member routes are only nested under `course`